### PR TITLE
Update module2.md

### PR DESCRIPTION
**Pull request by yuliaUU**

### CODE REVIEW

**Comment by yuliaUU** [comment]
I have issues with last question in Module 2 section 8 last question of the last quiz: 

The correct option there `The first and third options are both correct`.
I agree with the 1st option, but for 3rd option : in a manual I don't even see command `\`, `n\ or `s`: 

image

**Review by joelostblom — CHANGES_REQUESTED** [review]
One small suggestion.

Regarding the `less` commands, it is under "Searching" (and should also be in the slides, at least in the transcript)

image

File: chapters/en/module2.md [hunk 1]
```diff
@@ -268,7 +272,7 @@ A pipe works between commands, not between files.
 
 </choice>
 
-How can you navigate in long text pages (such as the manual) using <code>less</code>?
+Using the manual, look up what commands you can use to help you navigate in long text pages (such as the long word documents) using <code>less</code>?
```
[the remark this answers is not in the document]
Comment (yuliaUU) [reply chapters/en/module2.md:275]: Oh! Somehow my output cuts at line and I do not even see a Searching Section.. the image I showed is all I see..

[the remark this answers is not in the document]
Comment (joelostblom) [reply chapters/en/module2.md:275]: If you press `space` or `f` I think the page will scroll down

[the remark this answers is not in the document]
Comment (yuliaUU) [reply chapters/en/module2.md:275]: Yes. You are right! Less command is in a transcript!...

File: chapters/en/slides/module2/module2_05.md [hunk 2]
```diff
@@ -329,12 +329,14 @@ note_2022-07-02.md.bkp
 ```
 
 Notes:
+
```
Comment (joelostblom) [inline chapters/en/slides/module2/module2_05.md:332]:
```suggestion
```

Comment (joelostblom) [reply chapters/en/slides/module2/module2_05.md:332]: I'm making a nonsense change to trigger tests again

**Review by joelostblom — APPROVED** [review]

**Pull request merged**